src/Components/GameScreenComponents/Board.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Column } from "./Column"
+import "../../styles/GameScreen.css"
+
+function Board(){
+    return (
+        <div>
+            <div className="Game-board">
+                <div className="Column-Yaxis">
+                    <div>1</div>
```

Comment:
This component can be made much more flexible: instead of hard-coding a specific number of `div`s and `Column`s, you could define two constants `const NUM_ROWS = 6` and `const NUM_COLUMNS = 7` in a file `/src/constants.ts` and use these to automatically generate the right number of `div` and `Column` components